device/ospf/ospfv2/router.yaml

```diff
@@ -0,0 +1,171 @@
+          x-field-uid: 12
+    Ospfv2.Options:
+      description: |-
+        The OSPFv2 Options field is present Database Description packets and all LSAs.  
+        This enables OSPF routers to support (or not support) optional capabilities, 
+        and to communicate their capability level to other OSPF routers.
+        When capabilities are exchanged in Database Description packets a
+        router can choose not to forward certain LSAs to a neighbor because
+        of its reduced functionality.
+        Reference: A.2 The Options field: https://www.rfc-editor.org/rfc/rfc2328#page-46.
+      type: object
+      properties:
+        t_bit:
+          description: >-
+            Type of Service: 0th-bit: describes OSPFv2's TOS capability.
+          type: boolean
+          default: false
+          x-field-uid: 1
+        e_bit:
+          description: >-
+            External Capability: 1st-bit: describes the way AS-external-LSAs are flooded.
+          type: boolean
+          default: false
+          x-field-uid: 2
+        mc_bit:
+          description: >-
+            Multicast Capability: 2nd-bit: describes whether IP multicast datagrams are forwarded according to 
+            the specifications in [Ref18], rfc2328.
+          type: boolean
+          default: false
+          x-field-uid: 3
+        np_bit:
+          description: >-
+            NSSA Capability: 3rd-bit: describes the handling of Type-7 LSAs, as specified in [Ref19], rfc2328.
+          type: boolean
+          default: false
+          x-field-uid: 4
+        ea_bit:
+          description: >-
+            External Attribute: 4th-bit: describes the router's willingness to receive and forward External-Attributes-LSAs, 
+            as specified in [Ref20], rfc2328.
+          type: boolean
+          default: false
+          x-field-uid: 5
+        dc_bit:
+          description: >-
+            Demand Circuit: 4th-bit: describes the router's handling of demand circuits, as specified in [Ref21], rfc2328.
+          type: boolean
+          default: false
+          x-field-uid: 6
+        o_bit:
+          description: >-
+            Opaque LSA's Forwarded: 6th-bit: describes the router's willingness to receive and forward Opaque-LSAs, rfc2370.
+          type: boolean
+          default: false
+          x-field-uid: 7
+        unused_bit:
+          description: >-
+            Opaque LSA's Forwarded: 7th-bit: unused bit.
+          type: boolean
+          default: false
+          x-field-uid: 8
+    Ospfv2.GracefulRestart:
+      description: >-
+        Containter of properties of OSPFv2 Graceful Retstart.
+      type: object
+      properties:
+        enbale_helper_only:
+          description: >-
+            Support of Graceful Restart in Helper Mode. OSPF Restart Signaling (draft-nguyen-ospf-restart-06.txt).
+          type: boolean
+          default: false
+          x-field-uid: 1
+    Ospfv2.Advanced:
```

Comment:
Can we combine the two field here and previous attributes into a single OptionsBIts object ... why have just 2 here and rest in yet another object ?